guides/common/modules/proc_enabling-capsule-in-UI.adoc

```diff
@@ -0,0 +1,25 @@
+[id="assigning-organization-location-capsule-server_{context}"]
+
+= Enabling the {SmartProxyServer} in the {Project} web UI
+
+After installing the {SmartProxyServer} packages, you must enable the {SmartProxy} in the {Project} web UI by assigning the correct organization and location to it.
+
+.Procedure
+
+. Log into the {Project} web UI.
+. Click *Organization*, which is in the upper-left of the UI, then select *Any Organization*.
+. Click *Location*, which is in the upper-left of the UI, then select *Any Location*.
+. Click *Hosts* > *All Hosts* and select the checkbox next to the {SmartProxyServer}. 
+. Click the *Select Actions* list and select *Assign Organization*. 
+. Select organization from the *Organization* list, click *Fix Organization on Mismatch*, then click *Submit*. 
+. Select the checkbox next to the {SmartProxyServer}, click the *Select Actions* list, and select *Assign Location*. 
+. Select location from the *Location* list and click *Fix Location on Mismatch*, then click *Submit*.
+. Click *Administer* > *Organizations* and click on the organization name where you want to see the {SmartProxyServer}.
```

Comment:
```suggestion
. Navigate to *Administer* > *Organizations* and click the organization to which you have assigned {SmartProxy}.
```
From the IBM Style Guide:
Use “click,” not “click on,” to refer to the action of selecting something on a graphical interface.

Reply:
Fixed